### PR TITLE
Svelte check fixes for Column types

### PR DESCRIPTION
**Pull request by djbarnwal**
#1651

### CODE REVIEW

File: web-local/package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "check": "svelte-check --tsconfig ./tsconfig.json",
     "check:watch": "svelte-check --tsconfig ./tsconfig.json --watch",
     "lint": "eslint --ignore-path .gitignore .",
+    "svelte-check": "svelte-check",
```
Comment (ericpgreen2) [inline web-local/package.json:12]: We have `check` above, so this seems redundant

File: web-local/src/lib/components/column-profile/column-types/sparks/TimestampSpark.svelte [hunk 2]
```diff
@@ -41,14 +41,14 @@
...
+  // export let buffer = 4;
+  // export let leftBuffer = buffer;
+  // export let rightBuffer = buffer;
+  // export let topBuffer = buffer;
+  // export let bottomBuffer = buffer;
 
-  export let objectName: string;
-  export let columnName: string;
+  // export let objectName: string;
+  // export let columnName: string;
```
Comment (ericpgreen2) [inline web-local/src/lib/components/column-profile/column-types/sparks/TimestampSpark.svelte:51]: What's with all these comments? If the code isn't needed, can we just delete?

**Review by ericpgreen2 — APPROVED** [review]
Left a couple comments, but approving so you can merge once addressed. Nice to see a chunk of the `svelte-check` errors fixed!

File: web-local/src/lib/components/column-profile/column-types/sparks/TimestampSpark.svelte (hunk 2 above)
Comment (djbarnwal) [reply web-local/src/lib/components/column-profile/column-types/sparks/TimestampSpark.svelte:51]: The component as a whole is not being actively used anywhere but we might be using these sparklines inside the dimension table at some point. I would prefer to have to them as is for now as we might be refactoring and using these components in the next quarter.

Comment (ericpgreen2) [reply web-local/src/lib/components/column-profile/column-types/sparks/TimestampSpark.svelte:51]: Sure, sounds good.

**Pull request merged**